### ISSUE TITLE
[REQUEST] Selection modes

### ISSUE THREAD

**Issue by hlorus** [hlorus/CAD_Sketcher#40]
**Motivation / Existing Problem**
Native selection tools offer different modes, add the same to the addon's select tool.

**Description**
Add selection modes to allow adding/substracting/toggling selection. Idealy follow blender's default keymap, see: https://docs.blender.org/manual/en/latest/interface/selecting.html#selecting-modes

**Comment by amrsoll** [comment]
I guess we should shoot for this selection menu, like when using the Blender selection tool?
![image](https://user-images.githubusercontent.com/24782867/183714492-bd9cffc5-d544-4532-84d0-0d573300a24d.png)

It is not shown when selecting the Solvespace selection tool (yet)
![image](https://user-images.githubusercontent.com/24782867/183714654-6602b37c-9ad6-4788-be92-ed5add8ed8b1.png)

**Comment by hlorus** [comment]
> I guess we should shoot for this selection menu, like when using the Blender selection tool?
> ![image](https://user-images.githubusercontent.com/24782867/183714492-bd9cffc5-d544-4532-84d0-0d573300a24d.png)
> 
> It is not shown when selecting the Solvespace selection tool (yet)
> ![image](https://user-images.githubusercontent.com/24782867/183714654-6602b37c-9ad6-4788-be92-ed5add8ed8b1.png)
> 

Exactly! Although i'm not sure if the last two selection modes are really that useful...

**Comment by hlorus** [comment]
This is now supported, see:
https://github.com/hlorus/CAD_Sketcher/commit/8b2a71dd6917addfaa891e4c80c43c22054a688d

What's remaining are the shortcut bindings. The problem here is that shift is currently used to select through gizmos.

**Comment by hlorus** [comment]
The ctrl and shift modifiers are now also there, constraint selectability was moved to ctrl+shift.
